components/speedreader/rust/lib/src/readability/src/statistics.rs

```diff
@@ -61,7 +62,7 @@ const TEXT_LENGTH_SATURATION: usize = 1000;
 
 // For documents tagged as an article, only consider paragraphs with at least
 // 140 characters. This roughly corresponds to two English sentences.
-const PARAGRAPH_LENGTH_THRESHOLD_ARTICLE: usize = 140;
+// const PARAGRAPH_LENGTH_THRESHOLD_ARTICLE: usize = 140;
```

Comment:
Should we remove this line instead of commenting it out?

Reply:
fixed